fastchat/model/model_adapter.py

```diff
@@ -56,6 +56,14 @@
     "claude-instant-1",
 )
 
+COHERE_MODEL_LIST = (
+    "command",
+    "command-light",
+    "command-nightly",
+    "command-nightly-light",
+    "command-52b-v15-6-qa9cubq7-op-from-r2v6ya5f-p",
```

Comment:
Could you add a comment for reference to these model list?

Reply:
Done